data/upgrades.js

```diff
@@ -1800,6 +1810,18 @@
     "xws": "heavyscykinterceptor",
     "grants": [
       {
+        "type": "slot",
+        "name": "Cannon"
+      },
+      {
+        "type": "slot",
+        "name": "Torpedo"
+      },
+      {
+        "type": "slot",
+        "name": "Missile"
+      },
+      {
```

Comment:
Yeah, I haven't seen a lot of upgrade cards with choices like that. Maybe, we could add an `or` or `oneOf` property to the grant.  Might look something like:

```
    "grants": [
      {
        "oneOf": [
          {
            "type": "slot",
            "name": "Cannon",
          },
          {
            "type": "slot",
            "name": "Missle",
          },
          {
            "type": "slot",
            "name": "Torpedo",
          }
        ]
      }
    ]
```